### PR TITLE
Remove transition examples in docs

### PR DESCRIPTION
**Pull request by kennyt**
There are no more transitions in Tables

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @mlunoe

File: docs/src/Table/index.js [hunk 1]
```diff
@@ -117,16 +111,6 @@ class TableExample extends React.Component {
     this.forceUpdate();
```
Comment (daiyiwhale) [inline docs/src/Table/index.js]: Do we still need this?

Comment (kennyt) [inline docs/src/Table/index.js]: yeah this was unrelated to the transition i believe. this had something to do with the infinite scroll table and gemini scrollbar

Comment (daiyiwhale) [inline docs/src/Table/index.js]: Weird, okay :|

**Pull request merged**